scripts/player_audio_manager.gd

```diff
@@ -0,0 +1,38 @@
+extends Node
+
+@onready var landing_sfx: AudioStreamPlayer2D = $"Landing SFX"
+@onready var footstep_sfx: AudioStreamPlayer2D = $"Footstep SFX"
+@onready var jumping_sfx: AudioStreamPlayer2D = $"Jumping SFX"
+@onready var moth_jump_sfx: AudioStreamPlayer2D = $"Moth Jump SFX"
+# An array containing the valid surface types for sound effects. 
+@export var sfx_floor_materials: Array = ["dirt", "wood"]
+# The first item in the array is the default value. 
+@export var default_sfx_floor_material: String = sfx_floor_materials[0]
+@export var current_sfx_material: String = default_sfx_floor_material
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
+	set_sfx_floor_material(default_sfx_floor_material)
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta: float) -> void:
+	pass
+
+# Set the floor material beneath the player for triggering SFX. 
+# Invalid inputs will reset material to default_sfx_floor_material
```

Comment:
you should use a double # when making docstrings so it will show up with a hint when using it in other places in code.